lib/node_modules/@stdlib/math/base/assert/is-finitef/test/test.native.js

```diff
@@ -74,24 +74,3 @@ tape( 'the function returns `false` if provided `-infinity`', opts, function tes
 	t.equal( isfinitef( NINF ), false, 'returns false' );
 	t.end();
 });
-
-tape( 'the function returns `false` if not provided a finite number', opts, function test( t ) {
```

Comment:
Why was this test removed?

Reply:
As discussed on other similar PRs, this test was removed due to the add-on no longer explicitly checking for a number.

@DhruvArvindSingh for future PRs, please keep in mind that reviewers will not always have complete context. In which case, it can be very helpful if you use the pull request description to explain any salient changes. Removing tests is one such salient change. Providing a rationale for various changes helps reduce reviewer burden.